### ISSUE TITLE
Sidebar tables breaking the page

### ISSUE THREAD

**Issue by neilrooney** [reddit/reddit#1367]
Hi everyone. 

So I have had this bug for a while. I noticed it only happens on subreddits which make use of a table in their sidebar. For those that use a table, the page breaks i.e. I can move the page around to reveal white space on either side.

Some points: I am not using stylesheets on reddit, instead I'm using the default layout for all subreddits and I am also not using an adblocker. This is on a Mac 10.10.4 using Firefox.

**Comment by kjoconnor** [comment]
Is this on reddit.com or a local installation?

**Comment by neilrooney** [comment]
This is on reddit.com

**Comment by empyrical** [comment]
Which subreddits are causing you this problem? And which version of Firefox are you using?

**Comment by neilrooney** [comment]
@empyrical Any subreddit using a table in the sidebar e.g. r/IAmA. Firefox version is 39.0.

**Comment by 13steinj** [comment]
@neilrooney screenshot?

**Comment by neilrooney** [comment]
@13steinj Here you go: 

![screen shot 2015-07-29 at 21 24 25](https://cloud.githubusercontent.com/assets/10263660/8967638/63284d9a-3638-11e5-8991-44feeddfef20.png)
![screen shot 2015-07-29 at 21 24 59](https://cloud.githubusercontent.com/assets/10263660/8967639/634b5420-3638-11e5-8b33-d7e26a1bb9d8.png)

**Comment by 13steinj** [comment]
@neilrooney This is more of an issue with /r/IAMA then reddit, but reddit has it's own problems as well.

Firstly, those tables are made by the mods. If the width of the tables so happens to be larger than the sidebar, you get this page overflow. If you used /r/iama's stylesheet you wouldn't have this problem.

Secondly, to my knowledge, table width can't be set properly, and as such, it can't really have a msximum either without being dynamically done with less. I'll attempt to look into later and then submit a pull request, but I don't see this as reddit's issue.

**Comment by neilrooney** [comment]
@13steinj I disagree. r/IAMA is a part of reddit afterall all and like I previously mentioned, this is not only limited to this particular subreddit, instead it's a site wide problem on any subreddit which uses this type of table. I understand these tables are placed there by the mods but if reddit allows it's users to create them, which then causes the page to break in the form of a page overflow, then it is an important issue. 

I don't know about you but it is quite annoying when simply scrolling through pages.

**Comment by neilrooney** [comment]
Any update on this? It's driving me nuts.

**Comment by empyrical** [comment]
Does this issue show up for you when you're browsing on a different browser? Also, it looks like you've set some custom options like disabling css reddit-wide and disabling link flair. What settings do you have set?

If it can't be reproduced by anyone else it can't be fixed.

**Comment by 13steinj** [comment]
@empyrical I can reproduce it. I forgot about this until now, sorry @neilrooney .  I'll look into it now if there is a possible fix

**Comment by 13steinj** [comment]
@neilrooney Unfortunately, to my knowledge, because of the way tables work in web pages, the ability for one to alter it with CSS is mere impossible. With javascript; it would be theoretically possible, but the work outweighs the reward, since you would have to calculate font sizes in relation to both character length and the amount of columns. However [this](https://github.com/reddit/reddit/pull/1258) PR would technically fix it by putting any overflow on a horizontal scroll, which is the best that can be done. The PR seems to be inactive though.

**Comment by neilrooney** [comment]
@empyrical The issue is not present when using other browsers, at least no in Safari. I don't use Chrome. The bug can be reproduced as per @13steinj. I have tried different settings combinations with no luck. It is 100% a specific set of subreddits who use tables. r/science is particularly bad as they have four columns. See image: 
![screen shot 2015-08-17 at 14 23 39](https://cloud.githubusercontent.com/assets/10263660/9304775/08a8ab54-44ec-11e5-9dfb-40efa5d7bef9.png)

Here is a screenshot of all my settings: 
![screen shot 2015-08-17 at 14 24 05](https://cloud.githubusercontent.com/assets/10263660/9304754/d18cb908-44eb-11e5-98e8-1037238781d4.png)

Thanks @13steinj, I understand the issue. People are creating the tables with custom CSS in mind which is fine but they arent testing it when users dont wish to use the CSS. 

So where do we go from here? Can we make the table scrollable from left to right instead of the whole page for example?

**Comment by 13steinj** [comment]
@neilrooney Like I said:

> However #1258 should fix this would technically fix it by putting any overflow on a horizontal scroll


Which while it would not be automattically dynamically changing the tables, it would fix any page overflow (what you are experiencing) by putting a scollbar on the bottom of the sidebar if the side `<div>` has overflow. However I'm not an authourized contributor, as I'm not an admin, so I can't merge #1258 and you will just have to wait until it is considered.

**Comment by neilrooney** [comment]
Is it just me or has this problem been fixed? I can't replicate the issue any more using Firefox 41.0.2.

**Comment by neilrooney** [comment]
Nope, still broken!

**Comment by jkingsman** [comment]
This issue seems to be a spiritual duplicate of #1491 - same root cause with issues made for two different manifestations.
